RHEL/7/input/profiles/stig-rhel7-disa.xml

```diff
@@ -0,0 +1,752 @@
+<Profile id="stig-rhel7-disa">
+<title>STIG for Red Hat Enterprise Linux 7</title>
+<description>
+This profile aligns with the DISA STIG for Red Hat Enterprise Linux V1R1, released by DISA on xx-MARCH-2017.
```

Comment:
@shawndwells should we really have STIG version and release date in this profile? Seems like something else that would have to manually be updated which could be annoying.